Docs: Update private data plane docs (#837)
hasura/ddn-docs: docs/deployment/private-ddn/index.mdx

---
sidebar_position: 1
sidebar_label: Hasura DDN
description:
  "Get started with Hasura Private DDN to build secure, scalable, and customizable GraphQL APIs on your own
  infrastructure or in the cloud."

keywords:
  - hasura ddn
  - private ddn
  - byoc
  - vpc
  - enterprise
  - data plane
---

# Hasura Private DDN

## Introduction

> Hasura Private DDN allows you to provision private Data Planes — either on your own infrastructure, or in the cloud — to

Reviewer: Should we mention about provisioning in the cloud hosted and managed by Hasura?

Reviewer 2: Done!

> Hasura Private DDN allows you to provision private Data Planes — either on your own infrastructure, or in the cloud — to

Reviewer 3: ```suggestion
Hasura Private DDN allows you to provision private Data Planes — either on your own infrastructure, or dedicated Hasura infrastructure — to
```

give you complete control over your supergraphs. With Hasura Private DDN, enterprises can ensure data privacy, security,

> and compliance by hosting the data plane within their own Virtual Private Cloud (VPC) or any infrastructure you choose.

Reviewer 3: ```suggestion
and compliance by hosting the data plane within their own Virtual Private Cloud (VPC) provided by Hasura or any infrastructure you choose.
```
